learning/tour-of-beam/frontend/lib/pages/tour/screen.dart

```diff
@@ -19,33 +19,34 @@
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/svg.dart';
-import 'package:playground_components/constants/colors.dart';
 import 'package:playground_components/constants/sizes.dart';
+import 'package:playground_components/enums.dart';
 import 'package:playground_components/theme/color_provider.dart';
-import 'package:playground_components/widgets/split_view.dart';
+import 'package:playground_components/widgets/complexity.dart';
+import 'package:playground_components/widgets/split_view/controller.dart';
+import 'package:playground_components/widgets/split_view/widget.dart';
 
-import '../../components/borderless_expandable_wrapper.dart';
-import '../../components/complexity.dart';
+import '../../components/expansion_tile_wrapper.dart';
 import '../../components/page_container.dart';
 import '../../constants/sizes.dart';
 import '../../generated/assets.gen.dart';
 
-class MainScreen extends StatelessWidget {
-  const MainScreen();
+class TourScreen extends StatelessWidget {
+  const TourScreen();
 
   @override
   Widget build(BuildContext context) {
     return PageContainer(
       content: Row(
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          const _Summary(),
+          const _ContentTree(),
           Flexible(
             child: SplitView(
+              firstChild: const _Content(),
+              secondChild: const _Playground(),
+              controller: SplitViewController(isHorizontal: true),
```

Comment:
This will re-create the controller on every build and reset the ratio. Controllers must be in created in longer-living objects like `State` of a stateful widget or in other `ChangeNotifier`s. Yet another reason to do so is to be able to dispose them.

Reply:
Moved to [playground/frontend/playground_components/lib/src/widgets/split_view/widget.dart](https://github.com/akvelon/beam/pull/239/files#diff-1783ec2af4363b685f2314e28fa639aaa30466dd268c955fe9ff2b93ff224216)
Same problem there.